p1_x: -0.009575127349762185
p1_y: 0.8953424252237614
p1_z: 0.6318515828810215
q1_w: 0.01944670419555283
q1_x: 0.007878228213433447
q1_y: -0.8403380825984978
q1_z: 0.5416564096831384
p2_x: -0.03012625416818105
p2_y: -0.88186244692652
p2_z: 0.6295301628418514
q2_w: 0.5408134126829343
q2_x: -0.8403144206398445
q2_y: 0.01526861373260406
q2_z: -0.03404991279937766